lan867x/examples/lan867x_tcp_sockets/server/main/idf_component.yml

```diff
@@ -0,0 +1,9 @@
+## IDF Component Manager Manifest File
+dependencies:
+  ## Required IDF version
+  idf:
+    version: ">=4.1.0"
```

Comment:
Same as above. Supported IDF version will be probably depended on `MMD read/write` functions, if I'm not mistaken. Those are planned to be backported. However, it will take some time to get into official releases => latest IDF will be supported at the start and then create issue to track backports and update here once ready.

Reply:
Or can we just start with 4.4 (where we planning to backport) and drive it just by `lan867x` `idf_component.yml`?

Reply:
Went with the second method. Removed version check from both server and client examples.